server/src/main/java/org/opensearch/index/query/MatchBoolPrefixQueryBuilder.java

```diff
@@ -176,8 +176,10 @@ public String minimumShouldMatch() {
     }
 
     /** Sets the fuzziness used when evaluated to a fuzzy query type. Defaults to "AUTO". */
-    public MatchBoolPrefixQueryBuilder fuzziness(Object fuzziness) {
-        this.fuzziness = Fuzziness.build(fuzziness);
+    public MatchBoolPrefixQueryBuilder fuzziness(Fuzziness fuzziness) {
+        if (fuzziness != null) {
```

Comment:
I think we don't need `fuzziness != null` here, `null` is the default value for `MatchBoolPrefixQueryBuilder`, right?